forms/htmleditor/TinyMCEConfig.php

```diff
@@ -34,6 +34,7 @@ class TinyMCEConfig extends HtmlEditorConfig {
 		'remove_script_host' => true,
 		'convert_urls' => false, // Prevent site-root images being rewritten to base relative
 		'menubar' => false,
+		'language' => 'en',
```

Comment:
Just to clarify, this will be set to `i18n::get_tinymce_lang()` by `LeftAndMain->init()`. Which is a mapping of ISO locales to known tinymce language files, see `i18n::$tinymce_lang`. This functionality should probably be placed in `TinyMCEConfig`, but in general it works as intended